infrastructure/.vagrant/machines/airy-core/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:b2a9215c-db31-4f1b-b3db-1e3a1e879bbe
```

Comment:
I think the `.vagrant` folder is gone now because `minikube` ?

Reply:
yes! don't know how this snugged into my PR!